internal/controller/datadogagent/controller_reconcile_agent.go

```diff
@@ -176,7 +176,9 @@ func (r *Reconciler) reconcileV2Agent(logger logr.Logger, requiredComponents fea
 				overrideName = *componentOverride.Name
 			}
 		}
-		overrideFromProvider := kubernetes.ComponentOverrideFromProvider(overrideName, provider, providerList)
+		logger.Info("providerList", "providerList", providerList)
+		logger.Info("provider", "provider", provider)
```

Comment:
Can we switch these logs to a lower level (debug) ? Don't think they bring a lot of values in INFO
```json
{"level":"INFO","ts":"2025-08-26T09:26:34.734Z","logger":"controllers.DatadogAgent","msg":"providerList","datadogagent":{"name":"datadog-agent","namespace":"system"},"providerList":{"eks-abcdefg":{}}}
{"level":"INFO","ts":"2025-08-26T09:26:34.734Z","logger":"controllers.DatadogAgent","msg":"provider","datadogagent":{"name":"datadog-agent","namespace":"system"},"provider":"default"}
```